inc/classes/class-imagify.php

```diff
@@ -12,7 +12,7 @@ class Imagify {
 	 *
 	 * @var string
 	 */
-	const API_ENDPOINT = 'https://app.imagify.io/api/';
+	const API_ENDPOINT = IMAGIFY_APP_DOMAIN.'/api/';
```

Comment:
```suggestion
	const API_ENDPOINT = IMAGIFY_APP_DOMAIN . '/api/';
```

Reply:
this one generated an error in php cs
`Constant scalar expressions are not allowed when defining constants using the const keyword in PHP 5.5 or earlier.`
so I added new constant for API

https://github.com/wp-media/imagify-plugin/blob/5d82893ac318aae56da21bc3740469ec7dda4b17/imagify.php#L33